src/Frontend/Page/CreateIdea.hs

```diff
@@ -32,8 +32,8 @@ instance FormPageView PageCreateIdea where
 
     makeForm PageCreateIdea =
         ProtoIdea
-        <$> ("title"         .: DF.text Nothing)
-        <*> ("idea-text"     .: (Markdown <$> DF.text Nothing))
+        <$> ("title"         .: DF.text nil)
```

Comment:
I would use explicit `Nothing` here as we are not build abstraction over the monoid laws when we are dealing with form definition. It introduces mental juggling over types, and searching for the definition of `DF.text` .